django/core/management/commands/makemigrations.py

```diff
@@ -164,7 +164,7 @@ def handle(self, *app_labels, **options):
             if self.verbosity >= 1:
                 if len(app_labels) == 1:
                     self.stdout.write("No changes detected in app '%s'" % app_labels.pop())
-                elif len(app_labels) > 1:
+                elif app_labels:
```

Comment:
IMHO specifically this change worsen readability.

Something like this might look better:
```
if app_labels:
    if len(app_labels) == 1:
        ...
    else:
        ...
else:
    ...
```